docs/customer_management/cp_page_builder.md

```diff
@@ -0,0 +1,151 @@
+---
+description: Create unique Customer Portals for your clients with Page Builder.
+edition: experience
+---
+
+# Create Customer Portal
+
+On this page, you will learn how to configure the Customer Portal feature to be editable with Page Builder.
+If you already configured Customer Portal, you can learn how to build it with a Page Builder in [User Documentation.]([[= user_doc =]]/customer_management/build_customer_portal/) 
+
+## Configure Page Builder access to Customer Portal
+
+First, you need to create a root folder for Customer Portals, 
```

Comment:
Since https://github.com/ibexa/corporate-account/pull/175 it is also possible to mark `Customer Portal` CT as root, then only that one portal is accessible for given site access. But this is more like special case, not sure if it should be in that paragraph.